vmdb/lib/token_manager.rb

```diff
@@ -36,94 +36,42 @@ def self.configure(_namespace, options = {})
   end
 
   def self.gen_token(namespace, token_options = {})
-    token_cleanup
-    token = SecureRandom.hex(16)
-
     ts = global_token_store(namespace)
-    ts[token] = {:expires_on => Time.now.tv_sec + @config[:token_ttl]}
-    ts[token].merge!(prune_token_options(namespace, token, token_options))
+    token = SecureRandom.hex(16)
+    token_data = {:expires_on => Time.now.tv_sec + @config[:token_ttl]}
```

Comment:
the time object would just get serialized/deserialized into memcache like a normal object.  Then there's no need to convert or do anything fancy later on to "unconvert" it.  In other words, this code can just be `Time.now.utc + @config[:token_ttl]`